mythril/interfaces/cli.py

```diff
@@ -216,7 +216,7 @@ def create_parser(parser: argparse.ArgumentParser) -> None:
     options.add_argument(
         "--execution-timeout",
         type=int,
-        default=600,
+        default=86400,
```

Comment:
we can leave this line to the other PR.

Reply:
I reverted this so we can approve this pr faster. @b-mueller probably checked out a new branch while working on the timeouts